README.md

```diff
@@ -987,7 +987,7 @@ description: List of common variables that you can use to customize MegaLinter b
 
 | **ENV VAR**                                                                                                               | **Default Value**                             | **Notes**                                                                                                                                                                                                                                                                                                     |
 |---------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **ADDITIONAL_EXCLUDED_DIRECTORIES**                                                                                       | \[\]                                          | List of additional excluded directory basenames. they're excluded at any nested level.                                                                                                                                                                                                                        |
+| **ADDITIONAL_EXCLUDED_DIRECTORIES**                                                                                       | \[\]                                          | List of additional excluded directory basenames. They're excluded at any nested level.                                                                                                                                                                                                                        |
```

Comment:
All of the changes from these table, are they generated from the json schema or elsewhere, and the changes be made from there?